website_and_docs/content/blog/2023/selenium-4-11-0-released.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Selenium 4.11.0 Released!"
+linkTitle: "Selenium 4.11.0 Released!"
+date: 2023-07-31
+tags: ["selenium"]
+categories: ["releases"]
+author: Diego Molina ([@diegofmolina](https://twitter.com/diegofmolina))
+description: >
+  Today we're happy to announce that Selenium 4.11.0 has been released!
+---
+
+We're very happy to announce the release of Selenium 4.11.0 for Java, 
+.NET, Ruby, Python, and Javascript as well as the Grid and Internet Explorer Driver.
+Links to everything can be found on our [downloads page][downloads].
+
+### Highlights
+
+  * Chrome DevTools support is now: v113, v114, and v115 (Firefox still uses v85 for all versions)
+  * Support for [Chrome for Testing (CfT)](https://googlechromelabs.github.io/chrome-for-testing/) through Selenium Manager. Read more at the new Selenium Manager features [blog post](/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0/). 
+  * [Nightly builds](/downloads/#nightly) are being pushed for Ruby and Java. Support for other languages is coming soon.
+  * Deprecated all the code inside the Java `lift` package ([#12263](https://github.com/SeleniumHQ/selenium/pull/12263))
+  * Ignore process id match when finding the window handle - IE Mode on Edge. ([#12246](https://github.com/SeleniumHQ/selenium/pull/12246)) ([#12279](https://github.com/SeleniumHQ/selenium/pull/12279))
+
+
+#### Relevant improvements per language
+
+  * Java
+    * Make user defined SlotMatcher used everywhere in Grid code ([#12240](https://github.com/SeleniumHQ/selenium/pull/12240))
+    * Add support for FedCM commands ([#12096](https://github.com/SeleniumHQ/selenium/pull/12096))
```

Comment:
Do people know what "FedCM" is? I don't. A link to the spec might be useful: https://fedidcg.github.io/FedCM/

Reply:
Thank you for the reminded. I just added the link. We need to add docs for that.